opsdroid/testing/external_api.py

```diff
@@ -98,15 +99,12 @@ async def stop(self) -> None:
 
     async def _handler(self, request: web.Request) -> web.Response:
         route = request.path
-        if route in self._calls:
-            self._calls[route].append(request)
-        else:
-            self._calls[route] = [request]
-        if route in self._payloads:
-            self._payloads[route].append(await request.post())
-        else:
-            self._payloads[route] = [await request.post()]
-        status, response = self.responses[route].pop(0)
+        method = request.method
+
+        self._calls[(route, method)].append(request)
```

Comment:
I assume making this key a tuple instead of just route doesn't break a bunch of stuff downstream?

Reply:
I fixed all the things it broke :grinning: